internal/pkg/feature/cmd/feature/main.go

```diff
@@ -0,0 +1,275 @@
+package main
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"go/format"
+	"io/ioutil"
+	"os"
+	"strings"
+	"text/template"
+
+	"gopkg.in/yaml.v2"
+)
+
+const tmpl = `// Code generated by the feature package; DO NOT EDIT.
```

Comment:
Why are we templating this code? There seems to be only one instance of this file.

Reply:
This is the program that generates the code at `internal/feature/flags.go`. This file itself isn't used within the Flux library. It's invoked by `go generate`.